src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let total = 0;
+
+  const adder = function(number) {
+    if (number === undefined) {
```

Comment:
It is not necessary to compare exactly with `undefined`, you can simply check whether this argument exists through the logical NOT.